lib/puppet_litmus/rake_tasks.rb

```diff
@@ -171,7 +171,7 @@ def run_local_command(command)
     PE_RELEASE = 2019.0
     pe_latest_cmd = "curl http://enterprise.delivery.puppetlabs.net/#{PE_RELEASE}/ci-ready/LATEST"
     pe_latest = run_command(pe_latest_cmd, target_nodes, config: config_data, inventory: inventory_hash)
-    pe_latest_string = pe_latest[0]['result']['stdout'].gsub("\n", '')
+    pe_latest_string = pe_latest[0]['result']['stdout'].delete("\n")
```

Comment:
Would the `.strip` string function not be more suitable here?

Reply:
it was recommended by rubocop. so i'll go with it.